### PR TITLE
Switching auth + config to use IA s3 keys (instead of user/pass)

### PR DESCRIPTION
**Pull request by mekarpeles**
updated Open Library + client to use s3 keys if /account/login attempt json

### CODE REVIEW

File: olclient/cli.py [hunk 1]
```diff
@@ -61,18 +65,19 @@ def main():
     args = parser.parse_args()
 
     if args.configure:
-        username = args.username
-        if not username:
-            raise ValueError("--username required for configuration")
+        email = args.email
```
Comment (mekarpeles) [inline olclient/cli.py]: We might want to save the email in the config (it's useful for looking at the config values and seeing what account the s3 keys belong to)

**Comment by mekarpeles** [comment]
This PR dependent on https://github.com/internetarchive/openlibrary/pull/576

**Comment by mekarpeles** [comment]
Tests need to be updated to use s3 keys

File: CHANGES [hunk 2]
```diff
@@ -1 +1,2 @@
+v0.0.14, Tue Oct 3, 14:07:00 2017 -- Adds saving functionality.
```
Comment (cdrini) [inline CHANGES]: Update for 0.0.15 :)

**Pull request merged**